sql-database/monitor-and-scale-database/monitor-and-scale-database.sh

```diff
@@ -1,40 +1,55 @@
 #!/bin/bash

```

Comment:
This script fails due to `\r` (CR) tokens. Run `dos2unix` or a similar tool to change CRLF to LF.

Reply:
script succeeds for me - copy from my vscode into my bash shell - so, please explain

Reply:
figured this out, but where is this doc'd?

Reply:
https://review.docs.microsoft.com/en-us/search/?search=crlf&scope=Contributor&branch=master
no results

Reply:
when do we use LF and when do we use CRLF?

Reply:
Copy-paste is different from running scripts. You can control CRLF settings with git directly as described in the link in the checklist: https://help.github.com/en/articles/dealing-with-line-endings